app.py

```diff
@@ -26,12 +32,18 @@ def post():
             try:
                 request_input = request.get_json()
                 model_input = parse_input(request_input)
+                # print(model_input)
 
-                prediction = model.predict(model_input)
+                prediction = model.predict(np.array([model_input]))
```

Comment:
could you please convert to np.array in the parse_input fn itself to keep it clean here?